File: app/src/main/java/ashcollauthlogin/darkzek/com/CaptivePortalSystem/CaptivePortalFunctions.java
package ashcollauthlogin.darkzek.com.CaptivePortalSystem;

import android.content.Context;
import android.net.wifi.WifiInfo;
import android.net.wifi.WifiManager;

public class CaptivePortalFunctions {
    public static void IncreaseLoginCount(Context context) {
        //Get times logged in
        int timesLoggedIn = context.getSharedPreferences("credentials", 0).getInt("timesLoggedIn", 0);
        //Increase it by 1
        context.getSharedPreferences("credentials", 0).edit().putInt("timesLoggedIn", timesLoggedIn + 1).commit();
    }
}

File: app/src/main/java/ashcollauthlogin/darkzek/com/CaptivePortalSystem/CaptivePortalFunctions.kt
package ashcollauthlogin.darkzek.com.CaptivePortalSystem

import android.content.Context

object CaptivePortalFunctions {
    fun IncreaseLoginCount(context: Context) {
        //Get times logged in
        val timesLoggedIn = context.getSharedPreferences("credentials", 0).getInt("timesLoggedIn", 0)
        //Increase it by 1
        context.getSharedPreferences("credentials", 0).edit().putInt("timesLoggedIn", timesLoggedIn + 1).commit()
    }
}
